.circleci/config.yml

```diff
@@ -113,147 +125,108 @@ build_and_deploy: &build_and_deploy
-            git config --global user.name Dependabot
-          else
-            git config --global user.name $CIRCLE_USERNAME
-          fi
-          git config --global user.email $eGov_Email
-
           export COMMIT_MSG="$(git log -1 --pretty=%B)"
           git commit -m "CircleCI deployment for: " -m "$COMMIT_MSG" --quiet
 
           # When commit on master, we deploy to DEV
           if [[ $CIRCLE_BRANCH = "master" ]]; then
-            echo "Deploy to Pantheon DEV site"
-            # Set Pantheon environment name
             echo "export PANTHEON_ENV=dev" >> $BASH_ENV && source $BASH_ENV
-            # Set the connection type to GIT
-            terminus connection:set $CIRCLE_PROJECT_REPONAME.dev git
-            git push -f pantheon master
-            # TODO: Sanitize DB here
-            # terminus env:clone-content $CIRCLE_PROJECT_REPONAME.test dev
+          else
+            echo "export PANTHEON_ENV=$CIRCLE_BRANCH" >> $BASH_ENV && source $BASH_ENV
+          fi
+
+          # Check if the multidev branch site exists. If not, create it
+          terminus multidev:list --format=list --field=Name $CIRCLE_PROJECT_REPONAME > ./env_list.txt
+          if grep -Fxq "$PANTHEON_ENV" ./env_list.txt; then
+            echo "Found Pantheon site: $PANTHEON_ENV"
+          else
+            # Determine base branch of PR
+            #   - Extract PR number from pull request URL
+            #   - Query GitHub API for PR data and extract base branch from JSON base.ref property
+            [[ $CIRCLE_PULL_REQUEST =~ \/pull\/([0-9]+)$ ]]
+            BASE=$(curl -s https://api.github.com/repos/egovPDX/$CIRCLE_PROJECT_REPONAME/pulls/${BASH_REMATCH[1]} | grep -Pzo '(?s)"base":.*?"ref": "\K[A-Za-z0-9_-]+' | tr -d '\0')
+            if [[ $BASE = "master" ]]; then
+              BASE=dev
+            fi
+            # Create a new Multi-Dev with files and database from site corresponding to PR's base branch.
+            echo "Creating new multidev site cloned from $BASE. Please wait."
+            terminus multidev:create $CIRCLE_PROJECT_REPONAME.$BASE $PANTHEON_ENV
+          fi
+          rm -rf ./env_list.txt
+
+          # Set the connection type to GIT and force push
+          terminus connection:set $CIRCLE_PROJECT_REPONAME.$PANTHEON_ENV git
+          git push -f pantheon $CIRCLE_BRANCH
+          # Wait upto 240 seconds for the Sync code workflow to finish
+          echo "Waiting for the workflow: Sync code on $PANTHEON_ENV..."
+          terminus build:workflow:wait --max 240 $CIRCLE_PROJECT_REPONAME.$PANTHEON_ENV "Sync code on \"$PANTHEON_ENV\""
+
+          echo "Rebuilding cache..."
+          terminus drush $CIRCLE_PROJECT_REPONAME.$PANTHEON_ENV -- cr
+          echo "Running database updates..."
+          terminus drush $CIRCLE_PROJECT_REPONAME.$PANTHEON_ENV -- updb -y
+          echo "Importing config..."
+          terminus drush $CIRCLE_PROJECT_REPONAME.$PANTHEON_ENV -- cim -y -vvv
+          echo "Running cron..."
```

Comment:
I've removed the cron run on our deploys to test and prod because it often conflicted with the every 5 minute cron run on prod. Do we really need to run cron? `drush deploy` didn't include it so I felt comfortable removing.

Reply:
Cron removed